### PR TITLE
Fixes issue #2578

### LINKED ISSUE #2578: split term: MONDO_0016643 'frontonasal dysplasia'
**Mondo term (ID and Label)**
MONDO_0016643 'frontonasal dysplasia'

**Suggested revision and reasons**
This term has many children but the text def is:
Frontonasal dysplasia caused by mutations in the ALX3 gene, encoding homeobox protein aristaless-like 3. It is inherited in an autosomal recessive fashion.

I think the def should be revised to the Orphanet def:

A group of rare bone development disorders characterized by an array of abnormalities affecting the eyes, forehead, and nose, and linked to midfacial dysraphia. The clinical picture is highly variable, but the major findings include hypertelorism, a broad nasal root, a large and bifid nasal tip, and widow's peak. Occasionally, abnormalities can include accessory nasal tags, cleft lip, ocular abnormalities (coloboma, cataract, microphthalmia), conductive hearing loss, basal encephalocele and/or agenesis of the corpus callosum. Intellectual deficit is rare and more likely to occur in cases where hypertelorism is severe or where there is extra-cranial involvement.
Orphanet:250

And the NCIT term should be split into a new subclass.

### PR DESCRIPTION
**Pull request by rays22**
1. [x] Added database cross reference [NCIT:C129028](http://purl.obolibrary.org/obo/NCIT_C129028) as an annotation assertion of [frontorhiny MONDO:0007636](http://purl.obolibrary.org/obo/MONDO_0007636).
    - [x] Added annotation `source: MONDO:equivalentTo` to the dbxref annotation assertion.
2. Updated [`frontonasal dysplasia` MONDO:0016643](http://purl.obolibrary.org/obo/MONDO_0016643).
    - [x] Updated the definition with the `Frontonasal dysplasia` definition from [Orphanet:250](http://www.orpha.net/ORDO/Orphanet_250).
    - [x] Changed dbxref [NCIT:C129028](http://purl.obolibrary.org/obo/NCIT_C129028) annotation from `MONDO:equivalent` to `MONDO:superClassOf`
    - [x] Made synonym `frontonasal dysplasia 1` (dbxref NCIT:C129028) a *narrow* synonym.
    - [x] Made synonym `FND1` (dbxref NCIT:C129028) a *narrow* synonym.

Fix #2578

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -321171,18 +321172,18 @@ property_value: IAO:0000589 "meningioma (disease)" xsd:string
 [Term]
 id: MONDO:0016643
 name: frontonasal dysplasia
-def: "Frontonasal dysplasia caused by mutations in the ALX3 gene, encoding homeobox protein aristaless-like 3. It is inherited in an autosomal recessive fashion." [NCIT:C129028]
+def: "A group of rare bone development disorders characterized by an array of abnormalities affecting the eyes, forehead, and nose, and linked to midfacial dysraphia. The clinical picture is highly variable, but the major findings include hypertelorism, a broad nasal root, a large and bifid nasal tip, and widow's peak. Occasionally, abnormalities can include accessory nasal tags, cleft lip, ocular abnormalities (coloboma, cataract, microphthalmia), conductive hearing loss, basal encephalocele and/or agenesis of the corpus callosum. Intellectual deficit is rare and more likely to occur in cases where hypertelorism is severe or where there is extra-cranial involvement." [NCIT:C129028]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:321175]: Could you please change the definition source to Orphanet: Orphanet:250

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
Looks good, just one minor thing - the definition source should be changed to the Orphanet ID. thank you!

**Comment by nicolevasilevsky** [comment]
@rays22 - when you submit the PR, in the summary field, could you please write something that matches the issue (like update annotations 'frontonasal dysplasia' (MONDO:0016643), and then in the description field, all that info is great, but could you also add 'fix #[ticket number]', then it will link this PR to the ticket.

It has to be fix #[ticket number] (or address #[ticket number] or close #[ticket number]) (and those words can also be plural.) I don't think the tickets will be linked if you write fix issue #[ticket number] though.

Thanks!

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (rays22) [reply src/ontology/mondo-edit.obo:321175]: Done.

**Comment by rays22** [comment]
> @rays22 - when you submit the PR, in the summary field, could you please write something that matches the issue (like update annotations 'frontonasal dysplasia' (MONDO:0016643), and then in the description field, all that info is great, but could you also add 'fix #[ticket number]', then it will link this PR to the ticket.
> 
> It has to be fix #[ticket number] (or address #[ticket number] or close #[ticket number]) (and those words can also be plural.) I don't think the tickets will be linked if you write fix issue #[ticket number] though.
> 
> Thanks!

Thanks @nicolevasilevsky for catching the wrong definition source and your feedback on the PR messages. I will use the format as you suggest in the future.

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**